exchanges/gateio/gateio_websocket_option.go

```diff
@@ -295,14 +295,14 @@ func (e *Exchange) OptionsUnsubscribe(ctx context.Context, conn websocket.Connec
 }
 
 // WsHandleOptionsData handles options websocket data
-func (e *Exchange) WsHandleOptionsData(ctx context.Context, respRaw []byte) error {
+func (e *Exchange) WsHandleOptionsData(ctx context.Context, conn websocket.Connection, respRaw []byte) error {
 	push, err := parseWSHeader(respRaw)
```

Comment:
Shouldn't we add a check for the `conn` here? In-case a nil value of `conn` is provided

Reply:
Happy to panic here on that, all parameters will be provided by the websocket manager from its reader routine, if it is nil, something very wrong is very wrong. 🦇 🧛 🦇